nuxeo-retention-web/ftest/features/governance.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Retention Governance
+
+  As a Record Cleaner, I delete a document under legal hold in governance mode
+
+  Background:
+    Given user "John" exists in group "RecordManager"
+
+  Scenario: Record Cleaner Role
+    Given I have a File document
+    And "John" has ManageLegalHold permission on the document
+    When I login as "John"
+    And I browse to the document
+    Then I can see the "my document" document
+    When I set a legal hold on the document with description "My legal hold"
+    Then I see the document is under legal hold
+    And I cannot see the trash button
+    And I logout
+    Given I have the following groups
```

Comment:
Maybe we could add this part (https://github.com/nuxeo/nuxeo-retention/pull/84/files#diff-9686c656f24fad1331e01fc5f7cc7124d8bc398e7fdb6a3f700add128021e1b9R18-R25) to the scenario background, because it feels like the setup of the test (except the login part)

Reply:
(see example above)